plugins/plugin-radius/src/components/recipes/RecipeTable.tsx

```diff
@@ -41,12 +41,32 @@ export const RecipeTable = ({
     { title: 'Template Path', field: 'templatePath' },
   ];
 
+  const filters: TableFilter[] = [
+    {
+      column: 'Name',
+      type: 'select',
+    },
+    {
+      column: 'Type',
+      type: 'multiple-select',
+    },
+    {
+      column: 'Kind',
+      type: 'multiple-select',
+    },
+    {
+      column: 'Template Path',
+      type: 'multiple-select',
+    },
+  ];
+
   return (
     <Table
       title={title || 'Recipes'}
-      options={{ search: false, paging: false }}
+      options={{ search: true, paging: false, sorting: true }}
```

Comment:
since we have the TableFilter, we can turn of the search so that the toolbar disappears.

Reply:
I removed the search. I changed the filtering option for Name to multi-select to be consistent with the other options